app/helpers/validate/teachers.spec.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert')
+
+const teachers = require('./teachers')
+
+describe('teachers', function () {
+  it('should return empty list when has not errors', function () {
+    const disciplinas = [
+        {
+            teoria: {
+                name: 'Vanessa Kruth',
+                error: null,
+            },
+            pratica: {
+                name: 'Vanessa Kruth',
+                error: null,
+            }
+        },
+        {
+            teoria: {
+                name: 'Isidro',
+                error: null,
+            },
+            pratica: {
+                name: 'Isidro',
+                error: null,
+            }
+        },
+    ];
+
+    assert.equal(0, Array.from(teachers(disciplinas)).length)
+  })
+
+  it('should return list with one error', function () {
+    const disciplinas = [
+        {
+            teoria: {
+                name: 'Vanessa Kruth',
+                error: null,
+            },
+            pratica: {
+                name: 'Vanessa Kruth',
+                error: null,
+            }
+        },
+        {
+            teoria: {
+                name: 'Isidro',
+                error: 'Sala não encontrada',
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 16.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555382)
```suggestion
          error: 'Sala não encontrada',
```